chain-tx-enclave-next/mls/src/tree.rs

```diff
@@ -162,72 +173,66 @@ pub struct Tree {
 impl Tree {
     pub fn get_package(&self, leaf_index: LeafSize) -> Option<&KeyPackage> {
         match self.nodes[NodeSize::from_leaf_index(leaf_index).0] {
-            Node::Leaf(Some(ref pk)) => Some(pk),
-            _ => None,
+            Node::Leaf(ref kp) => kp.as_ref(),
+            _ => panic!("invalid node type"),
```

Comment:
why panic? what if it's an empty leaf?

Reply:
empty leaf will match the `Node::Leaf` pattern, it'll come to here only when node index and node type don't match.